dxp/demystifying_aui_input_tags.md

```diff
@@ -0,0 +1,727 @@
+# AUI Input
+
+## Table of Contents
+
+<div><a class="f5" href="#user-content-text">Text</a></div>
+<div><a class="f5" href="#user-content-textarea">Textarea</a></div>
+<div><a class="f5" href="#user-content-range">Range</a></div>
+
+The `<aui:input />` taglib is a custom tag used in `jsp` files to generate HTML form elements that conform to Liferay Portal's markup patterns, provide language translations, and other convenience methods. We should use this when adding form elements inside `jsp` files.
+
+The only required attribute is `name`, without it the `jsp` build will fail. The example below shows the minimum amount of code required for the taglib to output markup.
```

Comment:
This should be one of the following, to avoid the run-on sentence:

- "Then only required attribute is `name`, without which the `jsp` build will fail."
- "Then only required attribute is `name`; without it the `jsp` build will fail."